src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Windows.cs

```diff
@@ -84,12 +84,18 @@ public bool UseDefaultCredentials
             {
                 if (value)
                 {
+                    _winHttpHandler.DefaultProxyCredentials = CredentialCache.DefaultCredentials;
                     _winHttpHandler.ServerCredentials = CredentialCache.DefaultCredentials;
```

Comment:
Yes, the default for HttpClientHandler.UseDefaultCredentials is `false`.

https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```c#
        [Fact] 
        public void Ctor_ExpectedDefaultPropertyValues() 
        { 
            using (var handler = new HttpClientHandler()) 
            { 
                // Same as .NET Framework (Desktop). 
                Assert.False(handler.UseDefaultCredentials); 
            } 
        } 
```